core/templates/dev/head/pages/exploration_editor/editor_tab/teach_oppia_modal_directive.html

```diff
@@ -1,55 +1,66 @@
-</div>
+<style>
+.unresolved-answers-loading-text {
+  font-size: 1.17em;
+  text-align: center;
+}
 
-<div class="modal-body">
-  <div class="oppia-rule-details-header">
-    <test-interaction-panel state-content="stateContent" input-template="inputTemplate" on-submit-answer="submitAnswer(answer)">
-    </test-interaction-panel>
+.unresolved-answer {
+  border: black solid 1px;
+  border-radius: 5px;
+  padding: 1%;
+  margin-top: 1%;
+}
 
-    <div ng-if="trainingDataAnswer" style="padding-top: 10px; border-top: 1px solid #e5e5e5;">
-      <form name="testOppiaForm" class="form-inline">
-        <training-panel ng-if="classification.answerGroupIndex >= 0" answer="trainingDataAnswer" answer-feedback="trainingDataFeedback" answer-outcome-dest="trainingDataOutcomeDest" classification="classification" on-finish-training="finishTeaching(true)">
-        </training-panel>
-      </form>
-      <div ng-if="classification.answerGroupIndex === -1">
-        <div>
-          <span><strong>If Oppia encounters the answer:</strong></span>
-        </div>
+.unresolved-answer-feedback {
+  display: inline-block;
+  color: #00897B;
+}
 
-        <div>
-          <div angular-html-bind="answerTemplate">
-          </div>
-        </div>
+.unresolved-answer-response {
+  display: inline-block;
+  width: 75%;
+}
 
-        <br>
+.unresolved-answer-btn {
+  display: inline-block;
+  text-align: right;
+}
+</style>
 
-        <div>
-          <span><strong>it will reply with:</strong></span>
-        </div>
+<div class="modal-header">
+  <h3>Unresolved Answer</h3>
+</div>
 
-        <div>
-          <div angular-html-bind="trainingDataFeedback">
-          </div>
+<div class="modal-body">
+  <div class="unresolved-answers-list oppia-rule-details-header">
+    <p class='unresolved-answers-loading-text' ng-show="showLoadingDots">Loading <loading-dots></loading-dots></p>
+    <div ng-show="unresolvedAnswers.length > 0" style="padding-top: 10px; border-top: 1px solid #e5e5e5;">
+      <div class='unresolved-answer' ng-repeat="answer in unresolvedAnswers track by $index">
+        <div angular-html-bind="answer.answerTemplate">
         </div>
-
-        <br>
-
-        <div>
-          <span><strong>and then direct the learner to: </strong></span>
-          <span angular-html-bind="trainingDataOutcomeDest"></span>
+        <div class='unresolved-answer-response'>
+          <span><strong>Response</strong></span>
+          <span class='material-icons md-18'>arrow_right_alt</span>
+          <span class='unresolved-answer-feedback' angular-html-bind="answer.feedbackHtml"></span>
         </div>
-
-        <br>
-
-        <div>
-          <span><strong>This is due to a specific rule and therefore cannot be trained. If you do not think this is right, you should change the rule directly referring to this answer.</strong></span>
+        <div class='unresolved-answer-btn'>
+          <button type='button' class='btn btn-success' ng-click="confirmToTrainingData($index)">
```

Comment:
Ah. Maybe call it confirmAnswerAssignment() or similar?

Reply:
Done.